### DISCUSSION TITLE
Julia app updater error after upgrade to 1.9.2

### DISCUSSION THREAD

**Discussion by trdunsworth** [dat-linux/community#35] [Issues]
Using the app updater feature, I've started receiving an error: Plugin julia's list-all callback script failed with output:
after upgrading from 1.9.1 to 1.9.2

There is no additional output visible to the terminal screen.

**Comment by dat-linux** [comment]
Thanks for posting, they may have changed the endpoints. Will advise re solution.

**Reply by dat-linux** [reply]
Please run command **`cari update julia`** and retry install.

**Comment by trdunsworth** [comment]
Thank you. That worked very well.

On Wed, 2 Aug 2023 at 18:20, DAT Linux ***@***.***> wrote:

> Please run command *cari update julia* and retry install.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/dat-linux/community/discussions/35#discussioncomment-6621458>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAOBEW772H5N6ETKO4JB7LXTLHDZANCNFSM6AAAAAA275Z53Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
"A mind needs books as a sword needs a whetstone, if it is to keep its
edge."
- George R. R. Martin
